dt-core/admin/js/dt-options.js

```diff
@@ -12,69 +12,80 @@ jQuery(document).ready(function ($) {
 
   // Handle label language translations on a language
   window.update_language_translations = function (source, value) {
-    let translations = {};
+    console.log(source, 'line 17');
+    let translations_list = {};
     $(
       `#language_table .language_label_translations[data-field="${value}"]`,
     ).each(function (index, element) {
       const language = $(element).data('field');
-      if (!translations[language]) {
-        translations[language] = {};
+      if (!translations_list[language]) {
+        translations_list[language] = {};
+      }
+      if (!translations_list[language].translations) {
+        translations_list[language].translations = {};
       }
       const translation_key = $(element).data('value');
-      translations[language][translation_key] = $(element).val();
+      translations_list[language].translations[translation_key] =
+        $(element).val();
+    });
+    console.log(translations_list, 'line 26');
+
+    $.ajax({
+      type: 'POST',
+      dataType: 'json',
+      data: JSON.stringify(translations_list),
+      contentType: 'application/json; charset=utf-8',
+      url: `${window.dt_admin_scripts.rest_root}dt-admin-settings/languages/`,
+      beforeSend: (xhr) => {
+        xhr.setRequestHeader('X-WP-Nonce', window.dt_admin_scripts.nonce);
+      },
+      success: function (response) {
+        var languages = JSON.parse(response);
+        console.log(languages, 'success line 40');
```

Comment:
Nice work @CptHappyHands !

You can do `window.location.reload()` on the success for now.
Some in the other ajax